test/runnable/cppa.d

```diff
@@ -562,18 +562,22 @@ void test13289()
     assert(f13289_cpp_test());
 }
 
+version(Posix)
+{
+    enum __c_wchar_t : dchar;
```

Comment:
I get the error:
```
error : enum `Source.__c_wchar_t` forward reference of `__c_wchar_t.init`
```
When I try to:
```D
__c_wchar_t poo;
poo += 10;
```
Is because the enum has no body, it looks like a prototype.

Reply:
No there is a hack to allow no body, but `dchar`/`wchar` have a non-zero `.init`.

Reply:
Hmm, I can't seem to reproduce, I'll add that to the test suite and se what happens.

Reply:
I just tried this:
```d
enum __c_wchar_t : wchar;
void t()
{
    __c_wchar_t poo;
    poo += 10;
}
```
That does it for me...
```
error : enum `Source.__c_wchar_t` forward reference of `__c_wchar_t.init`
```

Reply:
Hmm, I get that if I run that on `run.dlang.io` but not if I run that locally with this PR.

Reply:
Is that what the `isSpecial` mischief is for? To resolve this issue?

Reply:
Silly question, but are you sure you are using this branch?

Reply:
Yes, it applies to `__c_long` and friends as well.

Reply:
Yeah no, I didn't spot that there was pre-existing special sauce being supplied; I was NOT on this branch. I just tried your code out of context.

Reply:
Called it!